data/sources/20190401-lastword.yaml

```diff
@@ -0,0 +1,32 @@
+title: The Last Word
+date: 2019-04-01
+
+topics:
+  -
+    categories: "Tax Policy"
+    direct_link: "https://youtu.be/N1Z6W5YZK1E?t=464"
+    tags:
+        - Trump's tax cuts
+        - Wealth Tax
+        - Financial Transactions Tax
+        - Marginal Tax Rates
+        - Corporate taxes
+  -
+    categories: "Democracy"
+    direct_link: "https://youtu.be/N1Z6W5YZK1E?t=624"
+    tags:
+        - Baby bonds
+        - Wealth Tax
+        - Other Democrat Perspectives    
+  -
+    categories: "Military"
+    direct_link: "https://youtu.be/N1Z6W5YZK1E?t=684"
+    tags:
+        - Leave Afghanistan
+        - Homeland
+  -
+    categories: "Personal"
+    direct_link: "https://youtu.be/N1Z6W5YZK1E?t=945"
+    tags:
+        - Pick top leadership
+        - Need diversity      
```

Comment:
`diversity` or `need for diversity` would work.

this answer as a whole is really interesting. the phrase that real struck out to me was `reflecting the people we serve` which we might want to include as a tag as well